web/themes/custom/cloudy/src/components/alert/readme.md

```diff
@@ -0,0 +1,22 @@
+# Alert
+
+## Props
+
+- `type`: string - info (default), danger, warning - Different types possible
+- `title`: string - The title of the alert
+- `content`: string - The body content of the alert
+- `isDraft`: boolean - If set to true, adds the `unpublished` badge
```

Comment:
The search_result component uses `is_not_published` as the prop. We should probably decide on a standard and make those consistent in all our components.